istio-control/istio-autoinject/values.yaml

```diff
@@ -16,6 +16,11 @@ sidecarInjectorWebhook:
   # even when mTLS is enabled.
   rewriteAppHTTPProbe: false
 
+  # If true, a self-signed CA will created in order to issue a certificate that
```

Comment:
Let's add a comment here that short term this is implemented using helm cert generation, and will migrate into istioctl.

Also mention that the cert private will be included in the helm template generation - so should only be used with iop install ( and make sure the iop doesn't save the template gen or removes it after ).

Reply:
Can you add a test ( in test/*.mk, to one of the targets ) ? Just to verify that a custom namespace can be used to install an injector, and maybe used.